src/liveevent/liveevent.controller.ts

```diff
@@ -0,0 +1,71 @@
+// eslint-disable-next-line no-redeclare
+import { Body, Controller, Delete, Get, Param, Post, Put } from '@nestjs/common';
+import { ApiOkResponse, ApiOperation } from '@nestjs/swagger';
+import { LiveEvent } from '@prisma/client';
+import { CreateEventDto, UpdateEventDto } from './dto/liveEvent.sto';
+import { LiveEventEntity, LiveEventWithIdEntity, SeparationEventListEntity } from './entity/liveEvent.entity';
+import { LiveeventService } from './liveevent.service';
+import { SeparationEventList } from './types';
+
+@Controller('liveevent')
+export class LiveeventController {
+   constructor(private readonly service: LiveeventService) {}
+
+   @Get()
+   @ApiOperation({ summary: '全件取得' })
+   @ApiOkResponse({ type: SeparationEventListEntity })
+   async getAllEvent(): Promise<SeparationEventList | {}> {
+      return this.service.getAll();
+   }
+
+   @Get('near')
+   @ApiOperation({ summary: '直近4件取得' })
+   @ApiOkResponse({ type: LiveEventEntity, isArray: true })
+   async getNearEvent(): Promise<LiveEvent[]> {
+      return this.service.getNearTime();
+   }
+
+   @Get(':date')
+   @ApiOperation({ summary: 'dateで取得' })
+   @ApiOkResponse({ type: LiveEventWithIdEntity })
+   async getEventBydate(@Param('date') date: string) {
+      return this.service.getByDate({ date });
+   }
+
+   @Get('/id/:id')
+   @ApiOperation({ summary: 'id指定で取得' })
+   @ApiOkResponse({ type: LiveEventWithIdEntity })
+   async getEvent(@Param('id') id: string): Promise<LiveEvent | []> {
```

Comment:
ここもidで取得できなかったら空配列が返ってくる？

```suggestion
   async getEvent(@Param('id') id: string): Promise<LiveEvent | []> {
```

Reply:
そう